CONTRIBUTING.md

```diff
@@ -45,6 +45,8 @@ The OSCAL project uses a typical GitHub fork and pull request [workflow](https:/
 1. Once you have staged your changes, you will need to commit them. When committing, you will need to include a commit message. The commit message should describe the nature of your changes (e.g., added new feature X which supports Y). You can also reference an issue from the OSCAL repository by using the hash symbol. For example, to reference issue #34, you would include the text "#34". The full command would be: ```git commit -m "added new feature X which supports Y addressing issue #34"```.
 1. Next, you must push your changes to your personal repo. You can do this with the command: ```git push```.
 1. Finally, you can [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+    - Please allow the NIST OSCAL developers to efficiently merge your pull with request. [Always allow edits from the maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
```

Comment:
Please review line #48. Suggested change:
"- Please allow the NIST OSCAL mainteners to make changes to your pull request, to efficiently merge it, by selecting on your fork [Allow edits from the maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)."

Reply:
s/mainteners/maintainers/

Reply:
Thanks for the feedback. Weirdly, I configured a spellcheck plugin into my IDE I used for this and it did not catch it, so that is frustrating and something I will have to check out. Thanks for the feedback to both of you, will merge these things in a bit.

Reply:
Updated in https://github.com/usnistgov/OSCAL/pull/1094/commits/b65b1f306426d5471ae30ea11e1f2394c9025819.